go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	k8s.io/client-go v0.18.2
 	sigs.k8s.io/controller-runtime v0.6.0
 )
+
+replace github.com/aws-controllers-k8s/runtime => /home/ubuntu/go/src/github.com/aws-controllers-k8s/runtime
```

Comment:
remove this and update after [runtime PR](https://github.com/aws-controllers-k8s/runtime/pull/6)

Reply:
You could also keep it in a `go.local.mod` just like elasticache controller => https://github.com/aws-controllers-k8s/elasticache-controller/blob/main/go.local.mod